config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
Because we will call the setting `MsBuildParameter-Windows` in the scripts and not as a default setting, the issue dotnet/buildtools#864 would be blocking this PR. The workaround is to create an alias for the `MsBuildParameter-Windows` in every command, but that is one of the things we want to avoid.
I'll make a priority to fix the issue in the Run Command Tool.